src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -2139,41 +2139,41 @@ public override DeclarationKind GetDeclarationKind(SyntaxNode declaration)
                     }
 
                 case SyntaxKind.VariableDeclaration:
-                {
-                    var vd = (VariableDeclarationSyntax)declaration;
-                    if (vd.Variables.Count == 1 && vd.Parent == null)
-                    {
-                        // this node is the declaration if it contains only one variable and has no parent.
-                        return DeclarationKind.Variable;
-                    }
-                    else
-                    {
-                        return DeclarationKind.None;
-                    }
-                }
-
-                case SyntaxKind.VariableDeclarator:
-                {
-                    var vd = declaration.Parent as VariableDeclarationSyntax;
-
-                    // this node is considered the declaration if it is one among many, or it has no parent
-                    if (vd == null || vd.Variables.Count > 1)
                     {
-                        if (ParentIsFieldDeclaration(vd))
+                        var vd = (VariableDeclarationSyntax)declaration;
+                        if (vd.Variables.Count == 1 && vd.Parent == null)
                         {
-                            return DeclarationKind.Field;
+                            // this node is the declaration if it contains only one variable and has no parent.
+                            return DeclarationKind.Variable;
```

Comment:
when do we have no parent...?

Reply:
That's only a whitespace diff. I haven't touched this. You should turn of whitespace in the diff settings.

Reply:
>That's only a whitespace diff. I haven't touched this. You should turn of whitespace in the diff settings.

Well... why did you change the whitespace ;-)  

You're right though.  Adding `?w=1` showed things better.  So i'm def ok with this change :)